src/index.d.ts

```diff
@@ -284,7 +286,7 @@ export namespace h {
 //
 // Preact render
 // -----------------------------------
-interface ContainerNode {
+export interface ContainerNode {
```

Comment:
Fairly odd: we reference this in `/compat/index.d.ts`, and that's always been fine, but now it's throwing an error, stating `ContainerNode` cannot be found. This seems pretty legit to me, seeing as how we weren't exporting it.